app/views/works/show.html.erb

```diff
@@ -60,11 +60,7 @@
   <% component.with_row(label: 'Deposit subtypes', values: [@work_presenter.all_work_subtypes]) %>
 <% end %>
 
-<%= render Elements::Tables::TableComponent.new(id: 'related-content-table', classes: 'mb-5') do |component| %>
-  <% component.with_caption do %><%= render Show::TableHeadingComponent.new(text: 'Related content', presenter: @work_presenter, tab: :related_content) %><% end %>
-  <% component.with_row(label: 'Related published work', values: @work_presenter.related_works) %>
-  <% component.with_row(label: 'Related links', values: @work_presenter.related_links) %>
-<% end %>
+<%= render Works::Show::RelatedContentsShowComponent.new(work_presenter: @work_presenter) %>
```

Comment:
Trivial but:

```suggestion
<%= render Works::Show::RelatedContentShowComponent.new(work_presenter: @work_presenter) %>
```

(I think we tend to refer to this as related content not related contents. 🤷🏻‍♂️)

Reply:
Makes sense. Fixing.